src/main/java/seedu/address/model/Model.java

```diff
@@ -78,7 +79,19 @@ public interface Model {
     void setPerson(Person target, Person editedPerson);
 
     /**
-     * Sorts all the persons in the addres book by the given {@code SortField}.
+     * Adds the given event.
+     * {@code event} must not already exist in the address book.
+     */
+    void addEvent(Event event);
+
+    /**
+     * Deletes the given event.
+     * The event must exist in the address book.
+     */
+    void deleteEvent(Event event);
+
+    /**
+     * Sorts all the persons in the address book by the given {@code SortField}.
```

Comment:
```suggestion
     * Sorts all the persons in the address book by the given {@code sortField}.
```
Hmm I'm unsure if this should be `sortField` or `SortField` as the previous JavaDoc used `event` instead of `Event` which refers to the object instead of the class.